spec/database_cleaner.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.configure do |config|
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, :js => true) do
```

Comment:
https://github.com/rubocop/ruby-style-guide#hash-literals

Reply:
Thanks for your response! I fixed it.